scripts/workers/code_upload_submission_worker.py

```diff
@@ -347,6 +347,11 @@ def main():
         message = evalai.get_message_from_sqs_queue()
         message_body = message.get("body")
         if message_body:
+            if (
+                challenge.is_static_dataset_docker_based_challenge
+                and not message_body.get("is_static_code_upload_submission")
+            ):
+                continue
```

Comment:
@savish28 let's not add these changes yet. We can these changes once we start working on the worker changes.